Whiskey/Tasks/Invoke-WhiskeyNspCheck.ps1

```diff
@@ -113,10 +133,21 @@ function Invoke-WhiskeyNspCheck
 
         Write-Timing -Message 'Running NSP security check'
 
+        $formattingArg = '--output'
+        if( !$nspVersion -or $nspVersion -gt (ConvertTo-WhiskeySemanticVersion -InputObject '2.7.0') )

```

Comment:
This should be `$nspVersion -ge (ConvertTo-WhiskeySemanticVersion -InputObject '3.0.0')`. Theoretically, they could release a 2.x or 2.7.x that continues to use the `--output` flag.

Reply:
Great point. I'll be updating the NSP check task to support the global WorkingDirectory property shortly. I'll commit this change too when I push it up.. :)